iron-list.html

```diff
@@ -1390,11 +1390,14 @@
 
       forceUpdate = forceUpdate || this._scrollHeight === 0;
       forceUpdate = forceUpdate || this._scrollPosition >= this._estScrollHeight - this._physicalSize;
+      // Probably a bug that style.height is a string like '100px' and it is being compared to
+      // a number?
       forceUpdate = forceUpdate || this.grid && this.$.items.style.height < this._estScrollHeight;
 
       // Amortize height adjustment, so it won't trigger large repaints too often.
       if (forceUpdate || Math.abs(this._estScrollHeight - this._scrollHeight) >= this._optPhysicalSize) {
         this.$.items.style.height = this._estScrollHeight + 'px';
+        this.updateViewportBoundaries();
```

Comment:
I have a repro case within the Chromium codebase new MD Settings page. Have not spent time to create a minimal repro case yet, but I'll do if you think this is useful.

In short, it happens when we render an iron-list while it is not visible (on purpose). Then the user navigates to the section that causes the iron-list to become visible and it only has 3 elements. Let me know if you think the time for a minimal repro is worth it.